<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadiationDosePage</class>
 <widget class="QWidget" name="RadiationDosePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="CTableView" name="tblRadiationDose"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblRecordCount">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Количество записей   </string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="CRadiationDoseSumLabel" name="lblDoseSum">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сумма доз</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="CPrintButton" name="btnRadiationDosePrint">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Печать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblActionSum">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Сумма количества действий   </string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblPhotosSum">
     <property name="text">
      <string>Сумма количества снимков    </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
  <customwidget>
   <class>CRadiationDoseSumLabel</class>
   <extends>QLabel</extends>
   <header>Events/Utils.h</header>
  </customwidget>
  <customwidget>
   <class>CPrintButton</class>
   <extends>QPushButton</extends>
   <header>library/PrintTemplates.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
